tests/backup/backup_rbac_test.go

```diff
@@ -499,3 +499,329 @@ var _ = Describe("{VerifyRBACforInfraAdmin}", func() {
 		log.FailOnError(err, "Unable to switch context to source cluster [%s]", SourceClusterName)
 	})
 })
+
+// VerifyRBACForPxAdmin Validates the RBAC operation for px-admin group user.
+var _ = Describe("{VerifyRBACForPxAdmin}", func() {
+	var (
+		scheduledAppContexts       []*scheduler.Context
+		adminBackupLocationName    string
+		adminBackupLocationUID     string
+		adminCloudCredentialName   string
+		adminCloudCredentialUID    string
+		scheduleNameMap            = make(map[string]string)
+		periodicSchedulePolicyName string
+		periodicSchedulePolicyUid  string
+		preRuleName                string
+		postRuleName               string
+		preRuleUid                 string
+		postRuleUid                string
+		adminBackupName            string
+		adminRestoreName           string
+		srcClusterUid              string
+		numOfUsers                 = 3
+		customRoleName             backup.PxBackupRole
+		userClusterMap             = make(map[string]map[string]string)
+		backupNameMap              = make(map[string]string)
+		userBackupNamesMap         = make(map[string][]string)
+		backupLocationMap          = make(map[string]string)
+		userNames                  = make([]string, 0)
+		providers                  = getProviders()
+		bkpNamespaces              []string
+		labelSelectors             map[string]string
+		mutex                      sync.Mutex
+	)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("VerifyRBACForPxAdmin", "Validates the RBAC operation for px-admin group user.", nil, 87887)
+		log.InfoD("scheduling applications")
+		scheduledAppContexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts := ScheduleApplications(taskName)
+			for _, appCtx := range appContexts {
+				appCtx.ReadinessTimeout = appReadinessTimeout
+				scheduledAppContexts = append(scheduledAppContexts, appCtx)
+				namespace := GetAppNamespace(appCtx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+
+	It("Validates the RBAC operation for px-admin group user.", func() {
+		Step("Validate applications", func() {
+			log.InfoD("Validating applications")
```

Comment:
Can we move it below ValidateApplicaitions?

Reply:
done.